E1_coldflow/brain_utils.h

```diff
@@ -182,7 +182,7 @@ void sensorReadFunc(int id) {
       batteryMonitor::readAllBatteryStats(farrbconvert.sensorReadings);
       break;
     case 4:
-      Thermocouple::Cryo::readCryoTemp(farrbconvert.sensorReadings);
+      Thermocouple::Cryo::readCryoTemps(farrbconvert.sensorReadings);
```

Comment:
The switch case is going to change soon here, just pointing it out to keep in mind